spec/models/video_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+require 'shoulda/matchers'
```

Comment:
That makes a lot of sense :) I'll amend the code.
Thanks for the suggestion and for taking time to look at my code, Matthew!